samples/v1.2/Scenarios/Agenda.json

```diff
@@ -17,7 +17,8 @@
 									"items": [
 										{
 											"type": "Image",
-											"url": "https://messagecardplayground.azurewebsites.net/assets/LocationGreen_A.png"
+											"url": "https://messagecardplayground.azurewebsites.net/assets/LocationGreen_A.png",
+											"altText": "Stop A"
```

Comment:
>Stop [](start = 23, length = 4)

Should we match this to reflect the image name vis LocationGreen_A ?

Reply:
I'd think that we can keep it simple as Location A. I don't know if we should use color names in alt text but I would guess we shouldn't